hooks/image/53-ek-content-preload

```diff
@@ -50,8 +50,14 @@ if [ -n "${EIB_KOLIBRI_CENTRAL_CONTENT_BASE_URL}" ]; then
   export KOLIBRI_CENTRAL_CONTENT_BASE_URL
 fi
 
+# Import all channel metadata and thumbnails for all channels
 for channel in $all_channels; do
   kolibri manage --skip-update importchannel network "${channel}"
+  EIB_RETRY_ATTEMPTS=2 EIB_RETRY_INTERVAL=30 eib_retry \
+    kolibri manage --skip-update \
+    importcontent --include-unrenderable-content --fail-on-error \
+    --node_ids="" --all-thumbnails \
+    network "${channel}"
```

Comment:
IMO, this should only be in the not full channels branch below. The full channels branch is obviously already going to import all the thumbnails. Not a big deal, though.